test-reports/lcov-report/index.html

```diff
@@ -0,0 +1,136 @@
+<!doctype html>
```

Comment:
@Emmanuelaaron I see you have unnecessary files that should actually be gitignored, for example, this one and all others in this `test-reports` directory. Kindly consider deleting this from the repository.

Reply:
I have resolved it @HabibSentongo.
Thank you for taking note of that